### PR TITLE
 Add seeker homepage

### LINKED ISSUE #182: Create a homepage for seekers
**Is your feature request related to a problem? Please describe.**
When seekers log in to the application and view their homepage, they should see the relevant apartments that the application has found.

**Describe the solution you'd like**
Create a seeker homepage

### PR DESCRIPTION
**Pull request by nadavsu**
# PR Dependencies
**Depends on #160
Depends on #176
Depends on #181**

# Description
- Created the front-end of the seeker's homepage.
- Enhanced the sidebar: added a user image and a welcome message
- Created apartment/roommate cards that load automatically with the page
- Routed the page to the backend of the web-app

## Screenshots
Screen Shot 2021-05-16 at 11 15 23

Screen Shot 2021-05-16 at 11 15 39

### Issues closed by this PR
Fixes #182

### CODE REVIEW

**Comment by tamirho** [comment]
WOW!!

**Review by Micha546 — COMMENTED** [review]
🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 
Looks GREAT!

**Comment by DanielMy777** [comment]
Good job!!
Looks like this needs a rebase so I'll review after that's done

**Comment by AmitAharoni** [comment]
Looking amazing!

**Review by AmitAharoni — APPROVED** [review]

File: main/views.py [hunk 1]
```diff
@@ -12,9 +13,7 @@ def index(request):
 @login_required
 def home(request):
     if request.user.is_seeker:
-        ''' TODO: delete current render and create a render_seeker_home
-            function that will render the homepage for the seeker.'''
-        return render(request, 'main/home.html')
+        return seeker_home(request)
     elif request.user.is_owner:
         ''' TODO: delete current render and create a render_owner_home
             function that will render the homepage for the owner.'''
```
Comment (tamirho) [inline main/views.py]: Please add here a call to the owner_home view

**Review by tamirho — CHANGES_REQUESTED** [review]

File: main/views.py (hunk 1 above)
Comment (tamirho) [reply main/views.py]: Nevermind.. resolved in #193

**Review by tamirho — APPROVED** [review]

**Comment by AmitAharoni** [comment]
@nadavsu 
Please rebase :)

**Pull request merged**